src/metal.cs

```diff
@@ -1893,10 +2220,18 @@ interface MTLRenderPassDescriptor : NSCopying {
 		[NoTV]
 		[Export ("renderTargetArrayLength")]
 		nuint RenderTargetArrayLength { get; set; }
+
+		[Mac (10,13)]
+		[Export ("setSamplePositions:count:")]
+		void SetSamplePositions ([NullAllowed] IntPtr positions, nuint count);
```

Comment:
OK, so unsafe code is not viable either (and as stated elsewhere, a managed array makes the generator bind thinking the native code is an NSArray).

I guess that leaves only `IntPtr`.